recipe/build.sh

```diff
@@ -7,10 +7,9 @@ cd build
 
 cmake ${CMAKE_ARGS} .. \
  -DCMAKE_BUILD_TYPE=Release \
- -DCMAKE_INSTALL_PREFIX=$PREFIX \
- -DCMAKE_POSITION_INDEPENDENT_CODE:BOOL=TRUE \
- -DCMAKE_INSTALL_LIBDIR=lib
+ -DBUILD_SHARED_LIBS=ON \
+ -DBUILD_SHARED_AND_STATIC_LIBS=ON || exit 1
 
-make -j $CPU_COUNT
+make -j $CPU_COUNT || exit 1
```

Comment:
No need for `exit 1`; with `set -e` this will stop anyways.
```suggestion
make -j $CPU_COUNT
```